.github/workflows/ci.yml

```diff
@@ -33,7 +33,7 @@ jobs:
 
       - uses: actions-rs/toolchain@v1
         with:
-          toolchain: nightly
+          toolchain: nightly-2023-06-27
```

Comment:
We may want to fix the compiler version for the whole project instead?

Reply:
right